provider/infoblox/infoblox.go

```diff
@@ -35,36 +35,46 @@ import (
 
 // InfobloxConfig clarifies the method signature
 type InfobloxConfig struct {
-	DomainFilter endpoint.DomainFilter
-	ZoneIDFilter provider.ZoneIDFilter
-	Host         string
-	Port         int
-	Username     string
-	Password     string
-	Version      string
-	SSLVerify    bool
-	DryRun       bool
-	View         string
-	MaxResults   int
+	DomainFilter      endpoint.DomainFilter
+	ZoneIDFilter      provider.ZoneIDFilter
+	Host              string
+	Port              int
+	Username          string
+	Password          string
+	Version           string
+	SSLVerify         bool
+	DryRun            bool
+	View              string
+	MaxResults        int
+	RecordsNameFilter string
 }
 
 // InfobloxProvider implements the DNS provider for Infoblox.
 type InfobloxProvider struct {
 	provider.BaseProvider
-	client       ibclient.IBConnector
-	domainFilter endpoint.DomainFilter
-	zoneIDFilter provider.ZoneIDFilter
-	view         string
-	dryRun       bool
+	client            ibclient.IBConnector
+	domainFilter      endpoint.DomainFilter
+	zoneIDFilter      provider.ZoneIDFilter
+	view              string
+	dryRun            bool
+	recordsNameFilter string
 }
 
 type infobloxRecordSet struct {
 	obj ibclient.IBObject
 	res interface{}
 }
 
+type byDNSName []*endpoint.Endpoint
+
+func (b byDNSName) Len() int      { return len(b) }
+func (b byDNSName) Swap(i, j int) { b[i], b[j] = b[j], b[i] }
+func (b byDNSName) Less(i, j int) bool {
+	return b[i].DNSName < b[j].DNSName
+}
+
 // MaxResultsRequestBuilder implements a HttpRequestBuilder which sets the
-// _max_results query parameter on all get requests
+// _max_results query parameter on all get requests or
```

Comment:
```suggestion
// _max_results query parameter on all get requests
```
Why is there an `or` here?